### ISSUE TITLE
Web editor not loading on Version 4.1.1

### ISSUE THREAD

**Issue by JonCris** [lucko/LuckPerms#791]
I just installed newest version on my test server but it throws this error when opening the editor:
```
[05:03:04 INFO]: [LP] Preparing a new editor session. Please wait...
[05:03:04 WARN]: java.lang.NoSuchMethodError: com.google.gson.JsonArray.add(Ljava/lang/String;)V
[05:03:04 WARN]: 	at me.lucko.luckperms.common.utils.gson.JArray.add(JArray.java:43)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.webeditor.WebEditor.lambda$formPayload$2(WebEditor.java:109)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.utils.gson.JArray.consume(JArray.java:61)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.webeditor.WebEditor.formPayload(WebEditor.java:107)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.commands.impl.misc.EditorCommand.execute(EditorCommand.java:95)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.commands.abstraction.SingleCommand.execute(SingleCommand.java:51)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.commands.abstraction.SingleCommand.execute(SingleCommand.java:43)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.commands.CommandManager.execute(CommandManager.java:201)
[05:03:04 WARN]: 	at me.lucko.luckperms.common.commands.CommandManager.lambda$onCommand$0(CommandManager.java:146)
[05:03:04 WARN]: 	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590)
[05:03:04 WARN]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[05:03:04 WARN]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[05:03:04 WARN]: 	at java.lang.Thread.run(Thread.java:745)
```

**Comment by ucraft02** [comment]
for me to
http://prntscr.com/ij2ds2

**Comment by merjilin** [comment]
I've managed to collect some more data from users in the discord guild.

- No sponge users have come forwarded with the issue and bungee users have said they have no issues, so it seems to be bukkit exclusive.
- The error seems to be occurring on 1.8.8 - those using 1.12.2 (There has been no users engaging in the conversation unless they're on these two versions).

**Comment by Logics4** [comment]
I tried to run LuckPerms-Bukkit-Legacy in a Paper 1.8.8 server, `/lp editor` works fine and gives me the link to the editor session. I guess it's because the Legacy version includes the missing libraries for 1.7.10 servers and gson is one of them.

So, (and also by reading the error, `java.lang.NoSuchMethodError`) it is likely a method that doesn't exist in 1.8.8.

**Comment by lucko** [comment]
Yep, you're both right.

Thanks :)

**Comment by ucraft02** [comment]
In 4.0.80 version work fine!!

**Comment by lucko** [comment]
Yeah, fix is included in `4.1.5` - which has now replaced the previous build on Spigot.

**Comment by ucraft02** [comment]
Thanks
